src/components/breadcrumbs/default.md.njk

```diff
@@ -0,0 +1,24 @@
+---
+layout: layout-example.njk
+---
+{% from "breadcrumb/macro.njk" import govukBreadcrumb %}
+
+{% macro code() %}
```

Comment:
It'd be really nice to get rid of the use of `macro()` if at all possible.

Reply:
Removed the macro from the example files.
We're reading the HTML markup from file and render it using the Nunjucks API.